history/characters/42000_high_elf.txt

```diff
@@ -53,19 +53,30 @@
 	properties="ai0e0a00000000"
 	culture=high_elf religion=cult_of_sunwell
 	martial=7 diplomacy=6 stewardship=6 intrigue=7 learning=8
-	add_trait=mastermind_theologian add_trait=ambitious add_trait=proud add_trait=stubborn add_trait=wroth add_trait=cynical
+	add_trait=mastermind_theologian
+	add_trait=content # Don't want to rule the elves
+	add_trait=just # Though he is not worthy to be the next king
```

Comment:
Are you sure that "though he is not worthy to be the next king" is reason for giving "just" trait, not "depressed"?

Reply:
He got depressed on the Frozen Throne branch.

Reply:
So... What you meant to say?
That you replaced just with depressed on the Frozen Throne branch or that he is both just and depressed on the Frozen Throne branch?

Reply:
Both.